2199/3/app/comment_analyzer.rb

```diff
@@ -0,0 +1,52 @@
+require 'dotenv/load'
+# analyze comment
+class CommentAnalyzer
+  ACCESS_KEY = ENV['ACCESS_KEY']
+  AZURE_ENDPOINT = ENV['AZURE_ENDPOINT']
+
+  def initialize(texts)
+    @texts = texts
+  end
+
+  # :reek:NestedIterators
+  # rubocop:disable Metrics/AbcSize
+  def analyze
+    JSON.parse(run_request.body)['documents'].each_with_object([]) do |result, store|
+      document = documents.find { |doc| doc[:id].to_s == result['id'] }
+      store << {
+        text: document[:text],
+        rating: result['score'] * 200 - 100
+      }
+    end
+  end
+  # rubocop:enable Metrics/AbcSize
+
+  def endpoint
```

Comment:
Again, what endpoint? I see from the code below that it's the `#azure_endpoint`. Why not say it explicitly?